cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/model/InstanceStoreMetadata.java

```diff
@@ -28,6 +28,11 @@ public Integer mapInstanceTypeToInstanceStoreCountNullHandled(String instanceTyp
         return instanceStoreCount != null ? instanceStoreCount : 0;
     }
 
+    public Integer mapInstanceTypeToInstanceSizeNullHandled(String instanceType) {
+        Integer instanceSize = instaceStoreConfigMap.getOrDefault(instanceType, VolumeParameterConfig.EMPTY).minimumSize();
```

Comment:
Did you test the functionality on Azure as well as AWS? AFAIK there are Azure instance types where besides the one ephemeral volume that comes with every Azure instance there are also additional NVMW ephemeral volumes that can be of different size than the default one ephemeral volume. And I think Azure implementation of the instance store metadata collection is not prepared for this scenario and only returns info about the one default ephemeral storage

Reply:
Yes, I had discussed this with @juhi-09. So, we had decided to remove this patch for AWS first and after that would look for changes in the Azure part.